Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -5623,4 +5626,11 @@ LayoutUnit RenderBox::intrinsicLogicalWidth() const
     return style().isHorizontalWritingMode() ? intrinsicSize().width() : intrinsicSize().height();
 }
 
+bool RenderBox::shouldIgnoreMinMaxSizes() const
+{
+    if (isFlexItem()) 
+        return downcast<RenderFlexibleBox>(parent())->isComputingFlexBaseSizes();
+    return false;
```

Comment:
Have you considered return isFlexItem() && downcast<RenderFlexibleBox>(parent())->isComputingFlexBaseSizes();